### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by vinaysbagale**
New Commit

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (vinaysbagale) [inline public/manifest.json:2]: Add some valid name

File: src/actions/constants.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  REGISTRATION: 'REGISTRATION',
+  generator: function (code, message, result) {
```
Comment (vinaysbagale) [inline src/actions/constants.js]: Please include comments

File: src/axios.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: 'http://localhost:6786/api'
```
Comment (vinaysbagale) [inline src/axios.js]: Create instances for staging/production

Comment (muddassarshaikh) [reply src/axios.js]: Added .env file for local env

**Pull request merged**